recipes/ocl/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "opencamlib" %}
+{% set version = "2019.07" %}
+{% set sha256 = "e08ab50672e24b51d30938ac60a6caa38bd8f5fb5f8fa3375dec0e69031cb620" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/aewallin/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [py<36]
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - ninja
+        - cmake 3.14
```

Comment:
can this be cmake >=3.14

Reply:
No, see referenced issue: https://github.com/conda-forge/cmake-feedstock/issues/91

Reply:
Well what about <=3.14? or !=3.15?

Reply:
Ok, this should work.